array/script.js

```diff
@@ -0,0 +1,97 @@
+var fruits = ['Apple', 'Banana'];
+var noMoreFruits = null;
+var notOnlyFruits = ['Apple', 'Banana', null,,,'Apple', false];
+var sameNumbers = [1, 2, 1, 3];
+
+console.log(isArray(fruits));
+console.log(isArray(noMoreFruits));
+console.log(range(5));
+console.log(range(1, 5));
+console.log(compact(notOnlyFruits));
+console.log(sum(range(5)));
+console.log(unique(sameNumbers));
+console.log(last(fruits));
+console.log(excludeLast(range(5)));
+console.log(excludeLast(range(5), 1));
+
+function isArray(obj){
+  return !!obj && obj.constructor === Array;
+}
+
+/* No 2
+function isArray(obj)
+{
+  return Object.prototype.toString.call(obj) === '[object Array]';
+}
+*/
+
+/* No 3
+function isArray(obj)
+{
+  return obj instanceof Array;
+}
+*/
+
+function range(a, b) {
```

Comment:
it gives wrong result with arguments (1, 5), (10, 15), (10, -5). when b is negative it will be infinite loop